### PR TITLE
Add a ValidateLogConfigs function to config.go.

### PR DESCRIPTION
**Pull request by pphaneuf**
This function is to validate a single-backend configuration.

### Checklist

- [ ] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: trillian/ctfe/config.go [hunk 1]
```diff
@@ -231,18 +231,62 @@ func BuildLogBackendMap(lbs *configpb.LogBackendSet) (LogBackendMap, error) {
 	return lbm, nil
 }
 
+func commonValidationLogConfigs(cfg []*configpb.LogConfig) error {
```
Comment (Martin2112) [inline trillian/ctfe/config.go]: That name is a bit ugly. Just `validateConfigs` or similar maybe?

Comment (pphaneuf) [reply trillian/ctfe/config.go]: Done.

**Comment by Martin2112** [comment]
Not sure why test coverage of configs is down almost 5%, might be spurious?

**Comment by pphaneuf** [comment]
Not sure what's up with that either? I did add a few lines that aren't tested, but 5%?!?

**Comment by pphaneuf** [comment]
The "-0.06% coverage" seems more accurate?

**Pull request merged**